MarketCollections.sol

```diff
@@ -167,66 +116,40 @@ contract MarketCollections {
       collectionId: internal Id of the collection
       <~~~*/
   /// @return Bool
-  function editMarketCollection(
-    bool[] memory isNotTradable,
+  function restrictMarketplaceContract(
+    bool[] memory restricted,
```

Comment:
This consolidates the 2 methods and is pretty clear IMO.

Reply:
This all looks good. Thankful for your watchful eye